paddlenlp/transformers/model_utils.py

```diff
@@ -306,7 +303,20 @@ def save_pretrained(self, save_directory):
                 model_config[key] = value.init_config
         with io.open(model_config_file, "w", encoding="utf-8") as f:
             f.write(json.dumps(model_config, ensure_ascii=False))
-        # save model
+
+    def save_pretrained(self, save_directory):
+        """
+        Save model configuration and related resources (model state) to files
+        under `save_directory`.
+        Args:
+            save_directory (str): Directory to save files into.
+        """
+        assert os.path.isdir(
+            save_directory
+        ), "Saving directory ({}) should be a directory".format(save_directory)
```

Comment:
"Saving directory ({}) should be a directory"
->
"`save_dir` ({}) is not available."